stone/target/js_client.py

```diff
@@ -24,6 +26,13 @@
           'contains all of the routes.'),
 )
 _cmdline_parser.add_argument(
+    'typedef_filename',
```

Comment:
Are you sure we should require this argument rather than generating additional type information only if a filename is specified?